tests/test_class_image_layer.py

```diff
@@ -34,6 +34,10 @@ def testInstance(self):
         self.assertEqual(self.layer.pkg_format, 'rpm')
         self.layer.os_guess = 'operating system'
         self.assertEqual(self.layer.os_guess, 'operating system')
+        self.assertFalse(self.layer.files_analyzed)
+        self.assertEqual(self.layer.files_analyzed, True)
```

Comment:
The way you do this is first set `self.layer.files_analyzed` to True and then `self.assertEqual` if it is True.

Reply:
@nishakm thanks for the review. Should I add `assertTrue` in place of `assertFalse`? or keep both of that?

Reply:
First do:
`self.layer.files_analyzed = True`
Then do:
`self.assertEqual(self.layer.files_analyzed, True)`

Reply:
I added the `assertFalse` due to this tests that was mentioned in issue: Test if the default value is False. Then `assertFalse` isn't required?

Reply:
Oh, no the assertFalse is still required and it works because we've set it to `False` by default and it works.
The second bit we need is to see whether we can toggle `files_analyzed` to `True`.

So first set it to true and then you can do the `assertEqual`.

BTW, you can also say `self.assertTrue(self.layer.files_analyzed)` just like the `assertFalse` statement. But the test will fail if you don't set it to `True` first ;)

Reply:
Ok, thanks!

Reply:
``` 
self.assertFalse(self.layer.files_analyzed)
self.layer.files_analyzed = True <-- please add this line
self.assertTrue(self.layer.files_analyzed)
self.assertEqual(self.layer.files_analyzed, True) <-- please remove this line
```

Explanation: All the tests in the `tests` folder are python unit tests. To learn more about what unit tests are, check out: https://en.wikipedia.org/wiki/Unit_testing. We want to test if we can assign `True` to this value and it does stay `True` after assigning. I hope it makes sense.

Reply:
@nishakm updated the PR. Thanks for sharing the link.